packages/@aws-cdk/aws-codecommit/test/test.codecommit.ts

```diff
@@ -42,8 +42,9 @@ export = {
     'fails when triggers have duplicate names'(test: Test) {
       const stack = new Stack();
 
-      const props = { repositoryName: 'MyRepository' };
-      const myRepository = new Repository(stack, 'MyRepository', props).notify('myTrigger');
+      const myRepository = new Repository(stack, 'MyRepository', {
+        repositoryName: PhysicalName.of('MyRepository'),
+      }).notify('myTrigger');
```

Comment:
`notify`? is this an API we like?

Reply:
Maybe not, but I don't feel like it's related to the physical name change.